ExeterTeam25/group-software-project
# Map appeareance updates

To test:

- Run the server
- Ensure map resolution is satisfactory
- Approve geolocation access and confirm new user marker appearance
- Enter challenges creation screen
- Ensure map resolution is satisfactory

Details:

Changed look of user marker and added shadow
Fixed tileset looking pixellated and blurry on maps
